### DISCUSSION TITLE
Got body.directives is not iterable after transitive dependency update of plugin-transform-block-scoped-functions

### DISCUSSION THREAD

**Discussion by davidmrnustik** [babel/babel#16940] [Q&A] [answered]
Hi there,
After an update of `@babel/plugin-transform-block-scoped-functions` to `v7.25.7`, when I run jest unit tests or build I got:
```
build	28-Oct-2024 10:17:36	[ERROR]     TypeError: /bamboo/../../*.js: body.directives is not iterable
build	28-Oct-2024 10:17:36	[ERROR] 
build	28-Oct-2024 10:17:36	[ERROR]       at callback (node_modules/@babel/traverse/src/path/introspection.ts:706:34)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.find (node_modules/@babel/traverse/src/path/ancestry.ts:37:9)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.isInStrictMode (node_modules/@babel/traverse/src/path/introspection.ts:682:30)
build	28-Oct-2024 10:17:36	[ERROR]       at transformStatementList (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:11:39)
build	28-Oct-2024 10:17:36	[ERROR]       at PluginPass.call (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:59:9)
build	28-Oct-2024 10:17:36	[ERROR]       at call (node_modules/@babel/traverse/src/visitors.ts:303:14)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call [as _call] (node_modules/@babel/traverse/src/path/context.ts:36:20)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call (node_modules/@babel/traverse/src/path/context.ts:21:18)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:97:31)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
```
During debugging session, I found out that some `body.directives` are `undefined` and as `undefined` is not iterable, an error is triggered.
Affected code line: https://github.com/babel/babel/blob/main/packages/babel-traverse/src/path/introspection.ts#L706

Unfortunately I cannot share production code. Also I'm not familiar with all the process of using dependencies `@babel/traverse` and `@babel/plugin-transform-block-scoped-functions` and where is triggered initially.

The only solution we have at the moment is to add `overrides` to `package.json`. But it's not ideal as many projects (more than 100) have to include it.
```
"overrides": {
    "@babel/plugin-transform-block-scoped-functions": "7.24.7",
    "@babel/preset-env": {
      "@babel/plugin-transform-block-scoped-functions": "7.24.7"
    }
  }
  ```

I'm using node v16.14.2 and npm v8.5.0.

Thank you in advance for any hint.

**Comment by JLHwung** [comment]
This seems to be a regression. Pending more information. A reproducible repo will definitely help

**Comment by Patrikoska1** [comment]
> Dobrý deň, po aktualizácii na `@babel/plugin-transform-block-scoped-functions`, `v7.25.7`keď spustím jest unit test alebo zostavím, som dostal:
> 
> ```
> build	28-Oct-2024 10:17:36	[ERROR]     TypeError: /bamboo/../../*.js: body.directives is not iterable
> build	28-Oct-2024 10:17:36	[ERROR] 
> build	28-Oct-2024 10:17:36	[ERROR]       at callback (node_modules/@babel/traverse/src/path/introspection.ts:706:34)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.find (node_modules/@babel/traverse/src/path/ancestry.ts:37:9)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.isInStrictMode (node_modules/@babel/traverse/src/path/introspection.ts:682:30)
> build	28-Oct-2024 10:17:36	[ERROR]       at transformStatementList (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:11:39)
> build	28-Oct-2024 10:17:36	[ERROR]       at PluginPass.call (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:59:9)
> build	28-Oct-2024 10:17:36	[ERROR]       at call (node_modules/@babel/traverse/src/visitors.ts:303:14)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call [as _call] (node_modules/@babel/traverse/src/path/context.ts:36:20)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call (node_modules/@babel/traverse/src/path/context.ts:21:18)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:97:31)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
> build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
> build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
> build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
> ```
> 
> Počas relácie ladenia som zistil, že niektoré `body.directives`sú `undefined`a keďže `undefined`to nie je možné opakovať, spustí sa chyba. Ovplyvnený riadok kódu: https://github.com/babel/babel/blob/main/packages/babel-traverse/src/path/introspection.ts#L706
> 
> Bohužiaľ nemôžem zdieľať produkčný kód. Tiež nie som oboznámený s celým procesom používania závislostí `@babel/traverse`a `@babel/plugin-transform-block-scoped-functions`tým, kde sa pôvodne spúšťa.
> 
> Jediné riešenie, ktoré momentálne máme, je pridať `overrides`do `package.json`. Nie je to však ideálne, pretože mnohé projekty (viac ako 100) to musia zahŕňať.
> 
> ```
> "overrides": {
>     "@babel/plugin-transform-block-scoped-functions": "7.24.7",
>     "@babel/preset-env": {
>       "@babel/plugin-transform-block-scoped-functions": "7.24.7"
>     }
>   }
> ```
> 
> Používam node v16.14.2 a npm v8.5.0.
> 
> Vopred ďakujem za akýkoľvek tip.

build	28-Oct-2024 10:17:36	[ERROR]     TypeError: /bamboo/../../*.js: body.directives is not iterable
build	28-Oct-2024 10:17:36	[ERROR] 
build	28-Oct-2024 10:17:36	[ERROR]       at callback (node_modules/@babel/traverse/src/path/introspection.ts:706:34)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.find (node_modules/@babel/traverse/src/path/ancestry.ts:37:9)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.isInStrictMode (node_modules/@babel/traverse/src/path/introspection.ts:682:30)
build	28-Oct-2024 10:17:36	[ERROR]       at transformStatementList (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:11:39)
build	28-Oct-2024 10:17:36	[ERROR]       at PluginPass.call (node_modules/@babel/plugin-transform-block-scoped-functions/src/index.ts:59:9)
build	28-Oct-2024 10:17:36	[ERROR]       at call (node_modules/@babel/traverse/src/visitors.ts:303:14)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call [as _call] (node_modules/@babel/traverse/src/path/context.ts:36:20)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.call (node_modules/@babel/traverse/src/path/context.ts:21:18)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:97:31)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitSingle (node_modules/@babel/traverse/src/context.ts:109:19)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:180:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitMultiple (node_modules/@babel/traverse/src/context.ts:99:17)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visit (node_modules/@babel/traverse/src/context.ts:178:19)
build	28-Oct-2024 10:17:36	[ERROR]       at traverseNode (node_modules/@babel/traverse/src/traverse-node.ts:40:17)
build	28-Oct-2024 10:17:36	[ERROR]       at NodePath.visit (node_modules/@babel/traverse/src/path/context.ts:104:33)
build	28-Oct-2024 10:17:36	[ERROR]       at TraversalContext.visitQueue (node_modules/@babel/traverse/src/context.ts:148:16)

**Comment by davidmrnustik** [comment, marked as answer]
Hello, just to confirm, that fix in https://github.com/babel/babel/pull/17024 did help to fix my issue. I'm using babel-core v7.26.7.
Thank you.
